services/frontend/src/sagas/api/location-api.js

```diff
@@ -1,4 +1,6 @@
 export function getCurrentLocation () {
```

Comment:
I don't see any reason why you are using `function` instead of `=>` to declare a function. Use modern syntax!